fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/navigation/CentralNavigationComponent.java

```diff
@@ -83,11 +87,22 @@ public class CentralNavigationComponent {
      */
     private Routing routing;
 
+
+    /**
+     * Public Transport routing
+     */
+    private PtRouting ptRouting;
```

Comment:
I asked myself the question why `PtRouting` couldn't implement `Routing`. Looking into the code this is obvious, but might also confuse others.

Maybe we rename `Routing` to `VehicleRouting` in line with the `CVehicleRouting` config?